module/VuFind/src/VuFind/Controller/AbstractSearch.php

```diff
@@ -868,7 +868,12 @@ public function facetListAction()
         $params->initFromRequest($this->getRequest()->getQuery());
         // Get parameters
         $facet = $this->params()->fromQuery('facet');
+        $contains = $this->params()->fromQuery('contains', '');
         $page = (int)$this->params()->fromQuery('facetpage', 1);
+        $ajax = (int)$this->params()->fromQuery('ajax', 0);
+        $ajaxLightbox = (int)$this->params()->fromQuery('ajaxLightbox', null);
```

Comment:
If you're making this an int, maybe default to 0 instead of null?

Reply:
It might also be helpful to add a comment explaining the difference between $ajax and $ajaxLightbox, because I don't think it's immediately obvious.